subprojects/gst-plugins-bad/sys/msdk/gstmsdkvpp.c

```diff
@@ -125,15 +134,16 @@ static GstStaticPadTemplate gst_msdkvpp_sink_factory =
     GST_PAD_ALWAYS,
     GST_STATIC_CAPS (GST_VIDEO_CAPS_MAKE (SUPPORTED_SYSTEM_FORMAT)
         ", " "interlace-mode = (string){ progressive, interleaved, mixed }" ";"
-        DMABUF_SINK_CAPS_STR));
+        DMABUF_SINK_CAPS_STR ";" VA_SINK_CAPS_STR));
```

Comment:
Here on Windows it will cause caps conversion failure because of the two consecutive ";". https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/2004 Please refer to this merge request to fix it thanks.

Reply:
According to the link above, do you mean we also need to add macro here, otherwise we will have "";"" for windows side which causes failure?

Reply:
Yes, on Windows DMABUF_SINK_CAPS_STR and VA_SINK_CAPS_STR are both empty strings, causing the caps be like "interlace-mode = (string){ progressive, interleaved, mixed } **; ;**" . The two ";" cannot be interpreted.

Reply:
We may add ";" to the caps_string on linux then we can avoid adding ";" at GST_STATIC_CAPS.